pkgs/top-level/all-packages.nix

```diff
@@ -27195,6 +27197,8 @@ in
 
   megaglest = callPackage ../games/megaglest {};
 
+  methane = callPackage ../games/methane {};
```

Comment:
```suggestion
  methane = callPackage ../games/methane { };
```

Reply:
is {<space>} rather than {} is that purely style, or does it actually mean something?